doc/how-to/sql/sql_tutorial.rst

```diff
@@ -30,7 +64,7 @@ Switch to the SQL language
 A feature of the client console program is that you can switch languages and
 specify the end-of-statement delimiter.
 
-Run the following commands to set the console input language to SQL and use
+Run the following commands to set the :ref:`console input <interactive_console_input_output>` language to SQL and use
```

Comment:
Looks like a prompt should looks smth like `sql_tutorial>`, below is the screen from the tt console for the `instance_scope` application:
<img width="429" alt="image" src="https://github.com/tarantool/doc/assets/38073144/dfd4d131-7c3e-441f-acf0-f321fbe39919">

Please take a look at other code snippets, too.

The same for MySQL, should be `mysql:instance001>` instead of `tarantool>`.

Reply:
Thanks! Fixed.